### ISSUE TITLE
How to remove stale files

### ISSUE THREAD

**Issue by malaybasu** [cyaninc/git-fat#61]
Is there anyway to remove stale files from git-fat?

**Comment by abraithwaite** [comment]
It's not a command in git-fat, but heres a start: `git fat status | sed '1,/Stale objects/d' | tr -d '\t'`

That'll list all the stale objects.

**Comment by malaybasu** [comment]
Thank you for the reply and pardon my ignorance. Can I just delete those objects from .git/fat/objects? Is there anything like original "git fat gc" command?

**Comment by abraithwaite** [comment]
You don't need a pardon, it's a good question!

You can delete them from `.git/fat/objects` if you're certain they've been pushed to the central repository.  Otherwise they'll be lost forever.

The git fat gc command was never reimplemented as a safety net to prevent people like myself from shooting themselves in the foot, but might still be useful with a fat warning message.

**Comment by malaybasu** [comment]
Thank you. Git history will still have a reference to original file, right? So if I revert back to a previous commit, the git fat file with the sha1 referece will come back, but the binary blob file will be missing. Will it create problem with git fat?

**Comment by abraithwaite** [comment]
You are correct entirely.  If you check out that commit again and the blob is missing locally, you can do another `git fat pull` and assuming it's on the server it will download it and restore it again. :-)
